velox/connectors/Connector.cpp

```diff
@@ -111,7 +111,8 @@ std::string commitStrategyToString(CommitStrategy commitStrategy) {
     case CommitStrategy::kTaskCommit:
       return "TASK_COMMIT";
     default:
-      VELOX_UNREACHABLE();
+      return fmt::format(
```

Comment:
What is the motivation for this change?

Reply:
It is good to know the exact bad enum number when this happens?